kernel/arch/dreamcast/include/dc/video.h

```diff
@@ -241,6 +227,16 @@ extern uint32 *vram_l;
 */
 int vid_check_cable(void);
 
+/** \brief   Set the VRAM convenience pointers.
+    \ingroup video_fb
+
+    This function sets the vram_s and vram_l pointers to specified offset
+    within VRAM. In multibuffered mode it allows manual management of them.
+
+    \param  base            The offset within VRAM to set the base to.
+*/
+void vid_set_vram(uint32_t base);
```

Comment:
```suggestion
void vid_set_vram(uintptr_t base);
```
Just talked about this one on Discord, wasn't sure what felt most idiomatic, but we agreed probably `uintptr_t` would actually be best here.

Reply:
Does that make sense when it isn't a pointer? It's only an offset within the VRAM not an exact memory location.

Reply:
Read the discussion you mentioned on your discord. Made sense. I believe the most appropriate thing would be for these to be pvr_ptr_t. Unfortunately there's a bit of a snarl where vid_set_start is used by PVR files and passed a uint32, and there are dozens of functions which treat those values and others that interact with them as uint32.

I'll make sure to take note of this though as a cleanup to be done within the PVR code to universally use pvr_ptr_t for addresses in/pointers into the vram.

Reply:
Sounds good to me, btw.